sadcowpng/sharedspaces/models.py

```diff
@@ -1,3 +1,52 @@
 from django.db import models
+from .forms import Noise_Level_Choices
 
-# Create your models here.
+
+# All information that will be sent to space
```

Comment:
nit: 'sent to space' is a bit vague.

Reply:
Hahaha, that actually funny. But yes I was saying we need to send our data to space. Clarified the statement.